plugins/org.eclipse.elk.core.service/src/org/eclipse/elk/core/service/LayoutMetaDataService.java

```diff
@@ -286,6 +286,11 @@ public void register(final LayoutAlgorithmData algorithmData) {
         @Override
         public void register(final LayoutOptionData optionData) {
             layoutOptionMap.put(optionData.getId(), optionData);
+            if (optionData.getLegacyIds() != null) {
+                for (String legacyId : optionData.getLegacyIds()) {
+                    layoutOptionMap.put(legacyId, optionData);
```

Comment:
I did not address every use case yet. With the current changes it should be possible to load existing models and layout them properly. Checking every usage of ``getId()`` and handling situations where the old properties are added explicitly remain to be taken care of.